quickstart.mdx

```diff
@@ -51,13 +51,15 @@ openai.Completion.create(
 // Make sure to `npm install promptlayer`
 import { promptlayer } from "promptlayer";
 
+// This assumes you already have `openai` installed
+// If not, you can install it with `npm install openai`
```

Comment:
Replace with:

// Make sure you have openai installed with `npm install openai`

Reply:
Done